workflow-steps/cache/output/post.js

```diff
@@ -5966,7 +5966,14 @@ function hashKey(key) {
   const globHashes = globsToHash.map((globPattern) => {
     return hashFileContents(globPattern);
   });
-  return [...hardcodedKeys, ...globHashes].join(" | ");
+  const hashCollections = [...hardcodedKeys, ...globHashes];
+  if (hashCollections.length > 1) {
+    return hash(hashCollections.join(" | "));
+  }
+  return hashCollections.join(" | ");
+}
```

Comment:
```suggestion
}

```

Reply:
these are auto-generated from the TS build 😢